### ISSUE TITLE
Suppress errors for forbidden resources

### ISSUE THREAD

**Issue by edjmao** [lensapp/lens#302]
**What would you like to be added**:
The ability to suppress error popups for forbidden resources, or simply the text "forbidden" or "insufficient access" where the resource is to be displayed.

**Why is this needed**:

We're trying to give read access to our development team into our Kubernetes cluster, but have restricted their ability to view secrets (like database passwords), and the initial feedback is that an error message pops up every time a pod is being viewed. The error reads:

secrets "<secret name>" is forbidden: User "<username>" cannoct get resource "secrets" in API group "" in the namespace "<ns name>"

We desire that the secrets stay hidden from the rest of the team. However, this error message gives the impression that there is something wrong with the configuration. It also becomes a distraction while poking around on the other resources in an otherwise great user experience.

**Environment you are Lens application on:**

- Kubernetes distribution: VMWare PKS (Tanzu)
- Desktop OS: Windows, MacOS, Linux

**Comment by edjmao** [comment]
Here's an example of a popup with the error. We'd like that "Secrets" field just have a "Forbidden" text rather than have the red popup show up every time a pod is loaded.

![image (3)](https://user-images.githubusercontent.com/17034455/80031368-3154b680-84af-11ea-9d08-1bf36a82b0b2.png)

**Comment by amitmahbubani** [comment]
+1, the problem is worse on the Secrets view when you have a larger number of namespaces :) 

<img width="828" alt="Screenshot 2020-05-15 at 2 32 13 AM" src="https://user-images.githubusercontent.com/5546730/81986135-e7499580-9654-11ea-9a82-7d86fc342baa.png">

**Comment by lucker6666** [comment]
quick fix, you can change one line in

/dashboard/client/components/notifications/notifications.tsx
```
@@ -16,7 +16,7 @@ export class Notifications extends React.Component {
   static ok(message: IMessage) {
     notificationsStore.add({
       message: message,
-      timeout: 2500,
+      timeout: 1,
       status: "ok"
     })
   }
```

all error message will not display.

**Warning: This will block all error message display !!!!**

**Comment by jakolehm** [comment]
Fixed in #366
